meson.build

```diff
@@ -17,11 +17,23 @@ config = configuration_data()
 config.set('PREFIX', get_option('prefix'))
 configure_file(input: 'config.h.in', output: 'config.h', configuration: config)
 
-version_h = vcs_tag(
-  command: ['git', 'describe', '--tags'],
+git_version = run_command('git', 'describe', '--tags', check: false).stdout().strip()
+if git_version == ''
+  git_version = 'devel'
```

Comment:
devel was not a good choice, it hide the fact that we don't have a tag. Using "unknown" like the git commit is a better choice.